.github/workflows/build.yaml

```diff
@@ -355,6 +363,12 @@ jobs:
         timeout-minutes: 200
         runs-on: macos-latest
         if: github.actor != 'restyled-io[bot]'
+        
+        env:
+            # NOTE: zap version should be kept in sync with the image used by 
+            # Docker, currently connectedhomeip/chip-build:0.6.31 
```

Comment:
This comment will need adjusting every time.  Can we remove the version number from here?  Same for other places with this comment.

Reply:
Updated to point to the script used to maintain these versions.

Reply:
Thank you!